flash/core/model.py

```diff
@@ -55,6 +56,11 @@ class Task(LightningModule):
         learning_rate: Learning rate to use for training, defaults to `5e-5`
     """
 
+    class Return(Enum):
```

Comment:
I like the idea, but the PR is outdate. Do you mind merging with master. Also, this logic should be handled by the postprocess object.

Reply:
hi @tchaton , as @carmocca pointed out that DataPipeline is completely revamped and codes were dependent on prev flow. I will create the logic again with postprocess object and update the PR.